packages/esm-sms-app/src/config-schema.ts

```diff
@@ -0,0 +1,52 @@
+import { Type } from '@openmrs/esm-framework';
+
+export const configSchema = {
+  configurationPageSize: {
+    _type: Type.Number,
+    _description: 'The default page size for the configuration table',
+    _default: 50,
+  },
+  deliveryTime: {
+    _type: Type.Array,
+    _description: 'The default delivery times for when testing configurations',
```

Comment:
> I'm not sure I understand this config option? Usually for `Array` types, each element is an object with defined properties.

Am trying to achieve customizable delivery time. I thought having them hardcoded might not be efficient.
<img width="745" alt="Screenshot 2024-08-22 at 17 55 05" src="https://github.com/user-attachments/assets/b34abb99-e53d-4347-a1dd-8bf1eb42b59a">

Reply:
So, in that case, I think this needs a better description, i.e., what values are valid, etc. It would also be useful to have a validator so that any erroneous values can at least be reported.